code/modules/mob/living/carbon/carbon.dm

```diff
@@ -140,6 +140,7 @@
 				stop_pulling()
 				if(HAS_TRAIT(src, TRAIT_PACIFISM))
 					to_chat(src, "<span class='notice'>You gently let go of [throwable_mob].</span>")
+					return

```

Comment:
doesn't aggressive grab->throwing people use a safe_throw_at() instead of a normal throw_at()? I think we should remove this pacifism check instead of fixing it, since I don't see why a pacifist shouldn't be allowed to throw someone nonharmfully.

Reply:
Throwing someone onto a glass table?